src/mainwindow.cpp

```diff
@@ -2687,7 +2687,7 @@ bool MainWindow::eventFilter(QObject *object, QEvent *event)
     }
     case QEvent::FocusOut:{
         if(object == m_searchEdit){
-            QString ss = QString("QLineEdit{ "
+            QString ss = QStringLiteral("QLineEdit{ "
```

Comment:
It will result in QString(QLatin1String("...") . QStringLiteral is the correct choice here.

Ref: https://github.com/KDE/clazy/blob/master/docs/checks/README-qstring-allocations.md

Reply:
Then why Qt doc is recommending to use QLatin1String ?

Reply:
@theshadowx If you read the docs carefully you'll notice that it recommends doing this only in the specific case where conversion to QString can be avoided. It gives string comparison as one example of this. However, the code here is assigning a value to a QString, in which case conversion to QString is unavoidable and it's better to create a QString at compile time, using QStringLiteral.

Reply:
Ok thanks for the clarification